cool-core/src/test/java/com/nus/cool/core/util/parser/CsvTupleParserTest.java

```diff
@@ -0,0 +1,54 @@
+package com.nus.cool.core.util.parser;
+
+import org.testng.Assert;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Paths;
+
+import java.util.Arrays;
+
+public class CsvTupleParserTest {
+
+    private String sourcePath;
+    @BeforeTest
+    public void setUp() {
+        System.out.println("Start To Test CvsTupleParser");
+        sourcePath = Paths.get(System.getProperty("user.dir"),
+                "src",
+                "test",
+                "java",
+                "com",
+                "nus",
+                "cool",
+                "core",
+                "resources").toString();
+    }
+
+    @Test
+    public void parseTest(){
+        String filepath = Paths.get(sourcePath, "parsertest", "table.csv").toString();
+        System.out.println(filepath);
+        CsvTupleParser csvParser = new CsvTupleParser();
+        try {
```

Comment:
Personally, directly throw exception in unitTest is better. No need to catch expection in unitTest

Reply:
sure, thanks.